migration/azure-fileshare-sa-deploy.json

```diff
@@ -0,0 +1,94 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountType": {
+            "defaultValue": "Premium_LRS",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS",
+                "Premium_LRS"
+            ],
+            "metadata": {
+                "description": "Storage Account type."
+            },
+            "type": "string"
+        },
+        "fileServerDiskSize": {
+            "defaultValue": "100",
```

Comment:
Wondering if we should default to 1024GB for max IOPS across the board. Thoughts @naioja ?

Reply:
make sense, and changed it to 1024 GB as default file server size.